### DISCUSSION TITLE
Add support to new PCI Video Trident TGUI-9440-3 or TGUI9680-1

### DISCUSSION THREAD

**Discussion by FigueiredoStable** [86Box/86Box#3880] [General]
Is possible to support this video card, i tryed to use Trident TGUI-9440AGi but with no success, my application is proprietary and i cant`t change the video card to another one.

tks

![Photo 295](https://github.com/86Box/86Box/assets/88445452/0df78e0c-4bf6-4c51-bbc1-ee8307d3be52)

**Comment by TC1995** [comment]
the 9440-3 and AGi are basically the same in details.

**Reply by FigueiredoStable** [reply]
Ok, I'll try to modify its configuration parameters.

Tks.
